administrator-guides/google-cloud/auto-translate/README.md

```diff
@@ -0,0 +1,42 @@
+# Google Auto Translate
+
+You can use Google's translation engine to translate every message received on a channel.
+
+For that you will need a Google Translation API key, you can find how to get one and pricing for it's usage from the [Official Google API documentation](https://cloud.google.com/translate/pricing).
+
+> Please note, that the Translation API is a paid feature from Google Cloud. If you decide to aquire this service, you will be paying Google and not Rocket.Chat
+
+## Getting the key
+
+Assuming that you already have a Google account logged, Follow these steps
+
+1. Go to <http://cloud.google.com/console/>
+2. Create a Project and select it.
+3. On the left side bar, Go to “APIs & auth” Tab => APIs Tab
+4. Enable “Translate API”
```

Comment:
Before number 4. please add:
4. Click on ENABLE APIS AND SERVICES and search for Cloud Translation API.